Components/Metrics/PCAMetric/itkPCAMetric_F_multithreaded.hxx

```diff
@@ -1084,7 +1084,7 @@ PCAMetric<TFixedImage, TMovingImage>::LaunchComputeDerivativeThreaderCallback()
 {
   /** Setup local threader and launch. */
   // \todo: is a global threader better performance-wise? check
-  auto local_threader = ThreaderType::New();
+  auto local_threader = MultiThreaderBase::New();
```

Comment:
Might use thread pools, so might _again_ improve the performance of PCAMetric ("PCAMetric_F_multithreaded") 👍